irohad/model/converters/impl/json_query_factory.cpp

```diff
@@ -18,7 +19,7 @@ using namespace rapidjson;
 namespace iroha {
   namespace model {
     namespace converters {
-      JsonQueryFactory::JsonQueryFactory(logger::Logger log)
+      JsonQueryFactory::JsonQueryFactory(logger::LoggerPtr log)
```

Comment:
`log_` is not used in this file. However, since this code is legacy, it is probably better not to refactor it.